.github/workflows/release.yml

```diff
@@ -32,6 +32,17 @@ jobs:
         uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b #v5.3.0
         with:
           python-version: "3.11"
+      - name: Install dependencies
+        run: pip3 install -r ./.scripts/requirements.txt
+      - name: Generate indexes
+        run: python3 ./.scripts/generate_indexes.py
+      - name: Commit Indexes
+        run: |
+          git config user.email "noreply@panther.com"
+          git config user.name "Panther Bot"
+          git add ./indexes
+          git commit -m "Update indexes"
+          git push
```

Comment:
Can we add the no-sign flag, like Christos did [here](https://github.com/panther-labs/panther-enterprise/pull/21058/files#diff-1b61d288c4a60eebc4b11a72b4936c0e689aed4dc84f50113a33a0af01e547caR50)?

Reply:
Thanks for taking a look. I've added the flag.